templates/keras_nlp/index.md

```diff
@@ -0,0 +1,89 @@
+# KerasNLP
+
+---
+## Quick links
+
+* [Developer guides](/guides/keras_nlp/)
+* [API reference](/api/keras_nlp/)
+
+---
+## Installation
+
+KerasNLP requires **Python 3.7+** and **TensorFlow 2.9+**.
```

Comment:
Do we need TF 2.9+ for KerasNLP? I know CV is waiting for the `BaseAugmentationLayer`.

Reply:
It has only been tested with 2.9 so this is a reasonable requirement.